backend/yugiohcardrarity/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+
+class YugiohCardRarity(models.Model):
+    rarity = models.CharField(
+        max_length=100,
+        unique=True,
+    )
+
+    class Meta:
+        verbose_name_plural = 'Yugioh Card Rarities'
+        verbose_name = 'Yugioh Card Rarity'
+
+    def __str__(self):
+        return {self.rarity}
```

Comment:
`__str__` method requires a string to be returned. In this case we are returning a set, which raises a TypeError upon creation. 

- [ ] Fix return value to be string

Reply:
fixed